### ISSUE TITLE
Vgg weights not normalized

### ISSUE THREAD

**Issue by mukund109** [anishathalye/neural-style#140]
The vgg weights have not been scaled as described in Section 2 of the [paper](https://www.cv-foundation.org/openaccess/content_cvpr_2016/papers/Gatys_Image_Style_Transfer_CVPR_2016_paper.pdf).

While this doesn't have an effect on the output, the style and content loss weights that I need to use to reproduce the results in the paper are different.

**Comment by mukund109** [comment]
![relu3_1](https://user-images.githubusercontent.com/15649806/53243316-68301080-36e2-11e9-93dd-89109adc9699.png)

**Comment by gpiosenka** [comment]
I ran this but iteration times were very large= 1000 iterations would take 48 hours? I have an RTX 2070 GPU which runs pretty fast. Wonder if you might have an advise as to way these times are so long.
Thanks

**Comment by wandadars** [comment]
@gpiosenka  I have the same card. I utilize the Nvidia NGC docker container to run this code and I can run something like 4000 iterations in 5 minutes.  I'm not sure why you are seeing such long runtimes for your hardware/software setup.

**Comment by anishathalye** [comment]
Yeah, the scaling is a bit different in the implementation here.

**Comment by gpiosenka** [comment]
I have run a lot of tensorflow and keras models on the GPU with very short run times so I do not know why this would be so slow even if was running on the cpu

**Comment by anishathalye** [comment]
I don't get what these comments have to do with the issue (vgg weights not being normalized).

In any case, it sounds like whatever you're doing is not actually using the GPU. Like @wandadars pointed out, it should run pretty fast. Are you feeding it gigantic images or something?
